src/01-simple-tests/index.test.ts

```diff
@@ -1,32 +1,43 @@
-// Uncomment the code below and write your tests
-// import { simpleCalculator, Action } from './index';
+import { simpleCalculator, Action } from './index';
 
 describe('simpleCalculator tests', () => {
   test('should add two numbers', () => {
-    // Write your test here
+    expect(simpleCalculator({ a: 5, b: 5, action: Action.Add })).toEqual(10);
```

Comment:
Best practice is not nesting functions in another function's params
```suggestion
    const args = { a: 5, b: 5, action: Action.Add };
    const result = simpleCalculator(args);
    const expectedResult = 10;
    expect(result ).toEqual(expectedResult );
```